contrib/typeshed/dill/_dill.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Any, Optional
+
+def dump(obj: Any, file: Any, protocol: Optional[Any] = ..., byref: Optional[Any] = ..., fmode: Optional[Any] = ..., recurse: Optional[Any] = ..., **kwds: Any) -> None: ...
+def dumps(obj: Any, protocol: Optional[Any] = ..., byref: Optional[Any] = ..., fmode: Optional[Any] = ..., recurse: Optional[Any] = ..., **kwds: Any): ...
+def load(file: Any, ignore: Optional[Any] = ..., **kwds: Any): ...
+def loads(str: Any, ignore: Optional[Any] = ..., **kwds: Any): ...
```

Comment:
Are all these `Any`s really needed? More specific type hints would be appreciated :-)